superfluid-finance/protocol-monorepo
# [ETHEREUM-CONTRACTS/SUBGRAPH/JS-SDK] | Issue #1220 | Migrate `scripts` to `ops-scripts` and `dev-scripts`

Issue Reference: https://github.com/superfluid-finance/protocol-monorepo/issues/1220

## High Level Changes
### Moving Scripts
- Moved almost all scripts to ops-scripts
- Moved deploy-test-environment to dev-scripts
- Updated workflows and bash scripts accordingly (scripts => ops-scripts)
- Updated documentation within `.js` scripts accordingly (scripts => ops-scripts)
- Update internal imports to ensure it doesn't break anything (subgraph)
- Update README.md in `ethereum-contracts` (scripts => ops-scripts)
- Update tsconfig's in `ethereum-contracts`
- Delete `index.js` in old `scripts`

cc @d10r
